starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/micronautGradle.rocker.raw

```diff
@@ -19,6 +27,16 @@ micronaut {
 @if(testRuntime != null) {
     testRuntime("@(testRuntime)")
 }
+@if(lambdaRuntimeMainClass != null) {
+    nativeLambda {
+@if (dsl == GradleDsl.KOTLIN) {
+        lambdaRuntimeClassName.set("@(lambdaRuntimeMainClass)")
+}
+@if (dsl == GradleDsl.GROOVY) {
+        lambdaRuntimeClassName = "@(lambdaRuntimeMainClass)"
+}
+    }
+}
```

Comment:
Do we need a test for this?

Reply:
https://github.com/micronaut-projects/micronaut-starter/pull/1948/commits/5ab6564a6db127f9546acd77bdc025861df6e82d